app/controllers/users/confirmations_controller.rb

```diff
@@ -51,7 +51,17 @@ def render_new_page
 
   def confirm_user
     @confirmable.confirm
+    notify_support_of_mew_user(@confirmable)
```

Comment:
Spelling of `new`/`mew`

Reply:
No need to pass in `@confirmable`, its available to the method below. Or is this a Clean Architecture style preference?